src/js/views/patients/worklist/layout.hbs

```diff
@@ -15,3 +15,4 @@
   <table class="worklist__header w-100 js-list-header" data-table-region></table>
 </div>
 <div class="flex-region list-page__list js-list" data-list-region></div>
+<div data-sidebar-region></div>
```

Comment:
I don't think this is needed.  I think the sidebar is one layout up from this

Reply:
You are right, removed that and everything still worked. Will be included in the next commit.